### PR TITLE
Add product page

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added Product Page

### CODE REVIEW

**Review by LordUttam — APPROVED** [review]
Looks good to me :) Add the author meta tag if you can.

File: stylesheets/product.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    width: 14rem;
+    top:0;
+}
+
+.category-filter{
+    display: flex;
+    flex-direction: column;
+    margin:1rem;
+    padding-bottom: 0.5rem;
+    border-bottom: 2px solid black;
```
Comment (aurobindoGupta) [inline stylesheets/product.css:28]: use var()

**Review by aurobindoGupta — APPROVED** [review]
good work

File: stylesheets/product.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+.rating-filter{
+    display: flex;
+    flex-direction: column;
+    margin:1rem;
+    padding-bottom: 0.5rem;
+    border-bottom: 2px solid black;
+}
```
Comment (xii-vi) [inline stylesheets/product.css:14]: you could use css combinator as both classes have the same property

**Review by xii-vi — APPROVED** [review]
_Go through the comment before merging. Also, add proper indentation._

**Comment by sah-shah7** [comment]
Went through the comments.  As PR approved, Merging

**Pull request merged**